apps/aechannel/src/aesc_fsm.erl

```diff
@@ -181,12 +183,12 @@ callback_mode() -> [state_functions, state_enter].
 %% |   |                 ^ |  ^ | |       channel_reestablish                 |   |
 %% |   |        update_* | |  | | |              |                            |   |
 %% |   |                 +-+  +-+ | shutdown     v                            |   |
-%% |   |                          |    +-------------+ [disconnect]           |   |
-%% |   |                          +--> |   closing   | ------------+          |   |
-%% |   |                               +-------------+             |          |   |
-%% |   |                                |  ^                       |          |   |
-%% |   |                                |  | channel_reestablish   |          |   |
-%% |   | <------------------------------+  |                       v          |   |
+%% |   |                          |    +------------------+   [disconnect]    |   |
+%% |   |                          +--> | mutual_closing   | ------+           |   |
```

Comment:
Update required in https://github.com/aeternity/protocol/tree/362d566af481214a5f650385b10d6cb3cfd0e831/channels#overview ?

Reply:
Yes, captured in [PT #163554702](https://www.pivotaltracker.com/story/show/163554702).